go/vm/geth/ct.go

```diff
@@ -190,33 +162,28 @@ func (i *callInterceptor) Call(env *geth_vm.EVM, me geth_vm.ContractRef, addr ge
 		kind = vm.StaticCall
 	}
 
-	var vmValue vm.Value
-	value.FillBytes(vmValue[:])
-
 	res, err := i.makeCall(kind, vm.CallParameter{
 		Sender:    vm.Address(me.Address()),
 		Recipient: vm.Address(addr),
-		Value:     vmValue,
+		Value:     value.Clone().Bytes32(),
```

Comment:
Is this a use case for `vm.Uint256ToValue`?

Also in the other functions below?